app/services/party_service.py

```diff
@@ -52,5 +55,9 @@ def get_user_details(uuid):
                          text=party_data.text,
                          url=url)
 
-            party_dict = json.loads(party_data.text)
-            return party_dict, party_data.status_code
+            if party_data.status_code == 200:
+                party_dict = json.loads(party_data.text)
+                return party_dict, party_data.status_code
+            else:
+                logger.error('Party service failed', status_code=party_data.status_code, text=party_data.text, uuid=uuid)
+                return party_data.text, party_data.status_code
```

Comment:
What should get returned if theres an error may need more thought. As this will now return a list for party_data.text (which is probably why there was the if statement above in the get_business_details function ^^). maybe discuss with Shane and Andrew on what the json should look like if details are not retrieved from party service.

Reply:
This shouldn't be an issue for for frontstage calls to sm (for now anyway). How appropriate this is depends on the form of error messages party service returns